tests/checkBoxes.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { test, expect } from '@playwright/test'
+
+test.describe('Checkboxes', async () => {
+	test.beforeEach(async ({ page }) => {
+		await page.goto('/')
+		await page.getByText('Veterinarians').click()
+		await page.getByText('All').click()
+		await expect(page.getByRole('heading')).toHaveText('Veterinarians')
+	})
+
+	test('Validate selected specialties', async ({ page }) => {
+		const selectedSpecialtyDd = page.locator('.selected-specialties')
+		const radiologyCb = page.locator('#radiology')
+		const surgeryCb = page.locator('#surgery')
+		const dentistryCb = page.locator('#dentistry')
```

Comment:
Don't make shortcuts in the naming like this. When I fist looked into the code I was like "what the heck is Cb"?

Always give a proper naming for constants, variables, and method names. You will thank yourself later :)